src/main.rs

```diff
@@ -0,0 +1,253 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_use(value_t)]
+extern crate clap;
+
+use clap::{App, AppSettings, Arg, SubCommand};
```

Comment:
Can you expand this into seprarate imports? I'm trying to avoid "nested" imports except where absolutely necessary.

Reply:
Done. Not sure why I forgot about those. Sorry.